app/config.py

```diff
@@ -22,6 +22,10 @@
         'name': 'pending',
         'order': 10,
     },
+    'paused': {
+        'name': 'paused',
+        'order': 10,
```

Comment:
What are the implications of having the same order as the `pending` status above?